content/services/services_ibm_z.rst

```diff
@@ -0,0 +1,29 @@
+IBM Z Development Hosting
+============================
+:slug: services/ibm-z
+:title: IBM Z Development Hosting
+:menu: Services, IBM Z Development Hosting, 4
+
+The Open Source Lab partners with `IBM`_ to host `IBM Z`_ based servers in order
+to provide an open platform for innovation to the open source community. With
+this hardware, the Open Source Lab offers continuous integration services via
+Jenkins allowing open source projects to build and test on the s390x architecture.
```

Comment:
Can you please also add this link: https://www.ibm.com/developerworks/community/groups/community/lozopensource as a source for additional information (similar to what we had on the power side?)